tagged-localbuild.xml

```diff
@@ -0,0 +1,774 @@
+  <project name="device-sony-voyager" path="device/sony/voyager" remote="sony" revision="674b0de906a8f9891b76e77f9b1ffc445b73e96f" upstream="r-mr1" dest-branch="r-mr1" groups="device,local::devices"/>
+  <project name="device-sony-yoshino" path="device/sony/yoshino" remote="sony" revision="a3411de18dbec7ec530023a486cf41a1185bdccb" upstream="r-mr1" dest-branch="r-mr1" groups="device,local::devices"/>
+  <project name="device/sample" revision="5c4f1d093d7771d6ac327e87e19698cec6bf3a34" upstream="refs/tags/android-11.0.0_r46" dest-branch="refs/tags/android-11.0.0_r46" groups="pdk"/>
+  <project name="exfatprogs" path="vendor/external/exfatprogs" remote="exfatprogs" revision="d9fbffd53e2e1ec89584b8c6931d8984e9894f4b" upstream="refs/tags/1.0.4" dest-branch="refs/tags/1.0.4" groups="device,local::external"/>
+  <project name="hardware-qcom-display" path="vendor/qcom/opensource/display/sm8150" remote="sony" revision="6428c9fabd2b8e80af7a689dec46bfa714627f7d" upstream="aosp/LA.UM.8.1.r1_r-mr1" dest-branch="aosp/LA.UM.8.1.r1_r-mr1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-display" path="vendor/qcom/opensource/display/sm8250" remote="sony" revision="0a7db0fdee114e2e9b7893fe2a5dc1bc7996d610" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-gps" path="vendor/qcom/opensource/gps" remote="sony" revision="0555e4fd41bbdac04f1d660d3d9b97946eb1731a" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-media" path="vendor/qcom/opensource/media/sdm660-libion" remote="sony" revision="c4da76d85056261a7cf6c78dc377d40b602f06ea" upstream="aosp/LA.UM.8.2.1.r1" dest-branch="aosp/LA.UM.8.2.1.r1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-media-sm8150" path="vendor/qcom/opensource/media/sm8150" remote="sony" revision="7c57b64bbaa26f5308177e5399956734a3300ff3" upstream="aosp/LA.UM.8.1.r1" dest-branch="aosp/LA.UM.8.1.r1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-media-sm8150" path="vendor/qcom/opensource/media/sm8250" remote="sony" revision="82fd2358795f5db13a9c21dc78b5d96304be2844" upstream="aosp/LA.UM.9.12.r1" dest-branch="aosp/LA.UM.9.12.r1" groups="device,local::qcom"/>
+  <project name="hardware-qcom-wlan" path="vendor/qcom/opensource/wlan" remote="sony" revision="6351125ec900f421276213a2a0d336cfc6268042" upstream="master" dest-branch="master" groups="device,local::qcom"/>
+  <project name="json-c" path="vendor/oss/json-c" remote="sony" revision="3b0d70d918bc57153721d45645cb953a493ebe19" upstream="master" dest-branch="master" groups="device,local::oss"/>
+  <project name="kernel" path="kernel/sony/msm-4.14/kernel" remote="sony" revision="838a06e04d8759ff5557ad93766748d30aabe72e" upstream="aosp/LA.UM.7.1.r1" dest-branch="aosp/LA.UM.7.1.r1" groups="device,local::LA.UM.7.1.r1"/>
```

Comment:
this, -common, and -headers repos for 4.14 are missing from localbuild.xml

Reply:
They were missing before should I really add them?

Reply:
You added them to tagged-localbuild.xml, so I assumed it's for 4.14 community devices that might be ported to Android 11:)

Reply:
That was accidental, but sure we can add them?

Reply:
Yes but only -common and -headers, please skip the kernel itself.